node/chainSimulator/components/storageService.go

```diff
@@ -35,5 +38,36 @@ func CreateStore(numOfShards uint32) dataRetriever.StorageService {
 		store.AddStorer(hdrNonceHashDataUnit, CreateMemUnit())
 	}
 
-	return store
+	if trieStoragePath.TriePath == "" {
+		return store, nil
+	}
+
+	config.AccountsTrieStorage.DB.FilePath = trieStoragePath.TriePath
+	storer, err := createStaticStorageUnit(config.AccountsTrieStorage)
+	if err != nil {
+		return nil, err
+	}
+
+	store.AddStorer(dataRetriever.UserAccountsUnit, storer)
```

Comment:
maybe add this section into a separate function, to avoid this early exit

Reply:
refactored